dagmc/dagnav.py

```diff
@@ -90,6 +90,25 @@ def geom_dimension_tag(self):
             create_if_missing=True,
         )
 
+    def add_volume_groups(self, group_map):
+        """Adds groups of volumes to the model.
+
+        Parameters
+        ----------
+        group_map : dict
```

Comment:
I'll suggest a slight change to the data structure here where the key is a tuple of group name and ID to keep the group information together in the key and the sets to be added as the value.